src/main/java/com/rakovets/course/java/core/practice/exception_handling/exception/NullException.java

```diff
@@ -0,0 +1,18 @@
+package com.rakovets.course.java.core.practice.exception_handling.exception;
+
+public class NullException extends NullPointerException {
```

Comment:
This isn't RuntimeException.
```suggestion
public class NullException extends RuntimeException {
```